lua/nvim-tree/git/runner.lua

```diff
@@ -82,9 +82,16 @@ function Runner:_run_git_job()
     pcall(uv.kill, pid)
   end
 
+  local function clean_log(t)
+    clean_tbl = vim.tbl_filter(function(v)
```

Comment:
This is a [stylua violation](https://github.com/kyazdani42/nvim-tree.lua/runs/7830009996).

You can simply return the result of tbl_filter.

Reply:
- function moved to `M.array_remove_nils(array)` within _lua/nvim-tree/utils.lua`
- stylua & luacheck should be good (I'm on Windows and am having issues with the hooks, so running manually)

Reply:
> I'm on Windows and am having issues with the hooks, so running manually

If you can fix the hooks I would be grateful; no nvim-tree developers have access or expertise with windows.